index.js

```diff
@@ -39,7 +39,8 @@ module.exports = {
 				'unicorn/error-message': 'error',
 				'unicorn/no-unsafe-regex': 'off',
 				'unicorn/prefer-add-event-listener': 'error',
-				'unicorn/prefer-exponentiation-operator': 'error'
+				'unicorn/prefer-exponentiation-operator': 'error',
+				'unicorn/no-console-spaces': 'error',
```

Comment:
You also need to add this to the readme, both in the example config and rule list.